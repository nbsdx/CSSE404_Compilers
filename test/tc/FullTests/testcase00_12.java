//Error case

class bar{

	public static void main(String[] args){
		int a = 4;
	}
}

class otherBar{
	public int bad(){
		return 1;
	}
}

class badBar extends otherBar{
	public bool bad(){
		return false;
	}
}