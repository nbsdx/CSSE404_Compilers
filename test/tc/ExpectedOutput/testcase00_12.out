Cannot find class named bool
Cannot overload methods.  Method bad has different type signature than inherited method of the same name.
Actual return type boolean of method bad does not match declared type bool
